Lombiq.Analyzers.PowerShell/Invoke-Analyzer.ps1

```diff
@@ -49,7 +49,7 @@ function Write-FileError([string] $Message, [string] $Path, [int] $Line = 0, [in
     if ($ForGitHubActions)
     {
         $Message = $Message -replace '\s*(\r?\n\s*)+', ' '
-        Write-Output "::error$(if ($Path) { " file=$Path,line=$Line,col=$Column::$Message" } else { "::$Message" })"
+        Write-Output "::error::$(if ($Path) { "$Path($Line,$Column): " })$Message"
```

Comment:
This now shows the absolute path in the runner, e.g. `/home/runner/work/Open-Source-Orchard-Core-Extensions/Open-Source-Orchard-Core-Extensions/Reset-Local.ps1`. Could we make it a relative path from the execution directory? In this case as just `Open-Source-Orchard-Core-Extensions/Reset-Local.ps1`.

This may be achieved using  `Resolve-Path -Relative -Path $Path`.

Reply:
I added this, so it now uses the relative path. See e.g.:

- https://github.com/Lombiq/PowerShell-Analyzers/actions/runs/9782938714/job/27010399747?pr=44#step:3:14
- https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/9782885486/job/27010468655?pr=804#step:3:14

Reply:
Very cool, thank you.